custom_components/tuya_local/devices/HCT-626_dual_water_timer.yaml

```diff
@@ -0,0 +1,197 @@
+name: Dual water timer
+products:
+  - id: key9nfnyp89j4esc
+  - id: ebb777wq1zxgr5og
+  - id: 8c63534780ffa00f
+primary_entity:
+  entity: valve
+  name: Valve 1
+  class: water
+  dps:
+    - id: 1
+      type: boolean
+      name: valve
+secondary_entities:
+  - entity: valve
+    name: Valve 2
+    class: water
+    dps:
+      - id: 2
+        type: boolean
+        name: valve
+  - entity: number
+    name: Timer 1
+    translation_key: timer
+    category: config
+    class: duration
+    dps:
+      - id: 17
+        type: integer
+        name: value
+        range:
+          min: 60
+          max: 3600
+        mapping:
+          - scale: 60
+            step: 60
+        unit: min
+  - entity: number
+    name: Timer 2
+    translation_key: timer
+    category: config
+    class: duration
+    dps:
+      - id: 18
+        type: integer
+        name: value
+        range:
+          min: 60
+          max: 3600
+        mapping:
+          - scale: 60
+            step: 60
+        unit: min
+  - entity: number
+    name: Weather delay remaining
+    category: diagnostic
+    dps:
+      - id: 42
+        type: integer
+        name: value
+        unit: hours
```

Comment:
Should make this read only. You can set it as-is but it doesn't seem to be respected.

Reply:
If read only, it should be a sensor with class duration. Unit should be h